angular-mentoring/src/app/coursesPage/courses/courses.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, EventEmitter, OnInit, OnChanges, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'courses',
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.css']
+})
+export class CoursesComponent implements OnInit {
+  @Input() courses: Array<Object>;
```

Comment:
Courses interface should be used here instead of Object

Reply:
I modified type of interface.

Reply:
Please use `Course` interface here instead of Object